surefire-booter/src/main/java/org/apache/maven/surefire/booter/PpidChecker.java

```diff
@@ -170,7 +169,7 @@ private void checkProcessInfo()
     // http://hg.openjdk.java.net/jdk7/jdk7/jdk/file/9b8c96f96a0f/test/java/lang/ProcessBuilder/Basic.java#L167
     ProcessInfo unix()
     {
-        ProcessInfoConsumer reader = new ProcessInfoConsumer( Charset.defaultCharset().name() )
+        ProcessInfoConsumer reader = new ProcessInfoConsumer( System.getProperty( "native.encoding", "UTF-8" ) )
```

Comment:
Why not a fallback fallback? native.encoding => file.encoding => UTF-8?

Reply:
Good idea. Somehow I didn't think of that, but it makes sense.